python/lsst/ts/rubintv/handlers/pages.py

```diff
@@ -55,16 +55,17 @@ async def get_home(
 ) -> Response:
     """GET ``/rubintv/`` (the app's external root)."""
     locations: list[Location] = request.app.state.models.locations
-    if len(locations) < 2:
```

Comment:
This was removed so that locations that just have one camera will now display the home page so that they have access to DDV too.